### ISSUE TITLE
[BABEL] Note: The code generator has deoptimised the styling of "ember-data/-private.js" as it exceeds the max of "100KB".

### ISSUE THREAD

**Issue by tsteuwer** [emberjs/data#5028]
Noticed this happening after removing our node_modules and reinstalling everything fresh:

```
[BABEL] Note: The code generator has deoptimised the styling of "ember-data/-private.js" as it exceeds the max of "100KB".
```
Happens on every `ember s`.

**Comment by stefanpenner** [comment]
@tsteuwer your most likely on an older version of ember-cli? Please share versions + reproduction

**Comment by stefanpenner** [comment]
https://github.com/emberjs/data/pull/5031 is a port of a fix that should make older versions of CLI not have ^. But without reproduction I am not 100% sure

**Comment by Driezzz** [comment]
In ember-cli-build.js add the following lines under "var app = "

```
var app = new EmberApp(defaults, {
            // Add these babel lines
            babel: {
                compact: false
            }
});
```

**Comment by xomaczar** [comment]
Fixed in 2.16.1
